frontend/packages/core/src/Table/table.tsx

```diff
@@ -166,20 +194,41 @@ const Table: React.FC<TableProps> = React.forwardRef(
           Filter out empty strings from column headers.
           This may be unintended which is why we override wit hthe hideHeader prop.
         */}
-          {showHeader && columns?.length !== 0 && columns.filter(h => h.length !== 0).length !== 0 && (
-            <MuiTableHead>
-              <StyledTableHeadRow>
-                {columns.map(h => (
-                  <StyledTableCell key={h} $responsive={responsive}>
-                    <Typography variant="subtitle3">{h}</Typography>
-                  </StyledTableCell>
-                ))}
-                {actionsColumn && !(responsive && compress) && (
-                  <StyledTableCell $responsive={responsive} $action />
-                )}
-              </StyledTableHeadRow>
-            </MuiTableHead>
-          )}
+          {showHeader &&
+            managedColumns?.length !== 0 &&
+            managedColumns.filter(h => h.title.length !== 0).length !== 0 && (
+              <MuiTableHead>
+                <StyledTableHeadRow>
+                  {managedColumns.map(h => (
+                    <StyledTableCell
+                      key={typeof h === "string" ? h : h?.id}
+                      $responsive={responsive}
+                      align="left"
+                      sortDirection={
+                        h?.sortable && defaultSort?.[0] === h?.id ? defaultSort[1] : false
```

Comment:
Just double checking but 'false' just means 'not sortable' right?

Just want to make sure that if it isn't set we don't have it turned on by default :)

Reply:
yes, false means that is not sortable